src/plugins/telemetryTable/components/table.vue

```diff
@@ -777,24 +789,29 @@ export default {
                     });
                     this.markedRows.splice(1);
                 }
-                let lastRowToBeMarked = this.visibleRows[rowIndex];
+                let lastRowToBeMarked = this.visibleRows[rowIndex],
+                    backwards;
```

Comment:
variable is set but not used anywhere that I can see?

Reply:
sorry, that was left over from an experiment. Thanks for catching